NEMO/views/consumables.py

```diff
@@ -49,27 +50,31 @@ def self_checkout(user) -> bool:
 @require_http_methods(["GET", "POST"])
 def consumables(request):
 	user: User = request.user
+	is_self_checkout = self_checkout(user)
 	if request.method == "GET":
 		from NEMO.rates import rate_class
 
 		rate_dict = rate_class.get_consumable_rates(Consumable.objects.all())
+		consumables_filter = Q(visible=True, allow_self_checkout=True) if is_self_checkout else Q(visible=True)
```

Comment:
Slight preference for avoiding Q() here. More consistent with the rest of the codebase
```suggestion
		consumable_list = Consumable.objects.filter(visible=True).order_by("category", "name")
		if is_self_checkout:
			consumable_list = consumable_list.filter(allow_self_checkout=True)
```